src/Api/SecretsManager/Models/Request/SecretCreateRequestModel.cs

```diff
@@ -32,4 +32,14 @@ public Secret ToSecret(Guid organizationId)
             Projects = ProjectIds != null && ProjectIds.Any() ? ProjectIds.Select(x => new Project() { Id = x }).ToList() : null,
         };
     }
+
+    public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
+    {
+        if (ProjectIds != null && ProjectIds.Length > 1)
```

Comment:
```suggestion
        if (ProjectIds is { Length: > 1 })
```

Reply:
Resolved in https://github.com/bitwarden/server/pull/2896/commits/8834331a623f06b28bb6fe8f030482dad65a9c15